src/reynir_correct/pattern.py

```diff
@@ -879,10 +920,20 @@ def wrong_verb_use(
         )
 
     def vera_að(self, match: SimpleTree) -> None:
-        start, end = match.span
-        text = "Mælt er með að sleppa 'vera að' og beygja frekar sögnina."
+        """ 'vera að' in front of verb is unneccessary """
+        # TODO don't match verbs that allow 'vera að'
+        # TODO exclude sentences where the subject is not a noun (and later, where not alive)
+        so = match.first_match("VP >> 'vera'").first_match("so")
+        nhm = match.first_match("TO > nhm").first_match("nhm")
+        start, _ = so.span
+        realso = match.first_match("IP-INF >> VP").first_match("so_nh")
+        _, end = realso.span
+        suggest = self.get_wordform(realso.tidy_text, realso.cat, so.all_variants, lemma=realso.lemma)
```

Comment:
`realso.cat` should always be `"so"`, right? If not, something is wrong and we should probably bail out.